test/Inputs/clang-importer-sdk/usr/include/macros.h

```diff
@@ -10,9 +10,9 @@
 #define GL_RGB 0x1907
 #define GL_RGBA 0x1908
 #define EOF (-1)
-#define UINT32_MAX 0xFFFFFFFFU
-#define INT64_MAX 0x7FFFFFFFFFFFFFFFLL
-#define UINT64_MAX 0xFFFFFFFFFFFFFFFFLL
+#define UINT32_MAX 0xFFFFFFFFu
+#define INT64_MAX 0x7FFFFFFFFFFFFFFFll
+#define UINT64_MAX 0xFFFFFFFFFFFFFFFFull
```

Comment:
That last one's now testing something different.

Reply:
Hmm, isn't `UINT64_MAX` supposed to be `unsigned`?

Reply:
C11 6.4.4.1p5: "The type of an integer constant is the first of the corresponding list in which its value can be represented. […] `ll` or `LL` – Octal or Hexadecimal Constant – `long long int`, `unsigned long long int`".

We happen to be getting the type from Clang at the moment, so there's no way Swift can screw this up, but if we ever rework that to manually check the suffix for some reason it's worth having a regression test about it.

Reply:
Alright, results in the imported types changing types as well :/  I think that we need to figure out how to have the conditional behavior checked then.

Reply:
This is conditional??

I guess we should check both `ll` and `ull`, then.

Reply:
Well, the value is used in a computation, which then results in type promotion rules kicking in, which results in type changes ... so much fun!